platforms/cortex/Makefile.include

```diff
@@ -1,8 +1,7 @@
 USEMODULE_INCLUDES_cortex := $(LAST_MAKEFILEDIR)/include
 USEMODULE_INCLUDES += $(USEMODULE_INCLUDES_cortex)
 
-CFLAGS += -DOBSTACLES_NUMOF=1
-CFLAGS += -DTRACEFD_NUMOF=1
```

Comment:
Why removing this CFLAGS ?

Reply:
Because in C++ I use std::list, so it is not necessary to define these sizes at compile time.